src/hatti/views/table.cljs

```diff
@@ -18,20 +18,22 @@
 (def pager-id "pager")
 
 ;; FIELDS
+(defn get-extra-fields
+   "Extra fields that will be displayed on the table."
+  [is-filtered-dataview?]
+  (let [extra-field  [{:full-name "_rank" :label "#" :name "_rank" :type "integer"}]
+        extra-fields (if is-filtered-dataview?
```

Comment:
No need to have this bound to a var in a let, since you're just going to return in